src/bundler.js

```diff
@@ -297,6 +301,13 @@ function processJsBundle(options, jsBundle, bundleDir, jsFiles, bundleName, cb)
 
     bundleStatsCollector.ClearStatsForBundle(jsBundle);
 
+    if (options.webpack) {
+        webpack.validate({

```

Comment:
This runs some simple checks to make sure it's a valid webpack bundle. Specifically it makes sure that only JS files are in the bundle (no ES6, no mustache, no JSX), and it makes sure that all files come in unminified/minified pairs (e.g. `file1.js` & `file1.min.js`).